dds/DdsDcpsCore.idl

```diff
@@ -300,6 +300,25 @@ module DDS {
 
     // ----------------------------------------------------------------------
 
+	  typedef unsigned long ParticipantLocation;
```

Comment:
These additions should all be in an `OpenDDS` IDL module instead of `DDS` (probably just `OpenDDS::DCPS` but others are OK as well)